CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+- [Ruby] Updated rubocop and rubocop gems (RSpec/Rake/Performance), also updated project files affected by the updated rules ([#342](https://github.com/cucumber/cucumber-expressions/pull/342))
```

Comment:
This is not necessary. Notable in this context means "notable to the user" and "notable for the purposes of semver".

Reply:
Yeh, ideally here you'd maybe write a quick changelog that you autofixed some styles or something. As some of the changes could break, but this would go into a patch release if it was released

Reply:
Sorry, but not completely sure if I should keep the changelog entry or remove it 😅

Reply:
I think what Rien is saying is technical things about ruby don't belong in a polyglot changelog (This is a library with 6 flavours).

So maybe just say somethinig similar to what has been written prior

Reply:
Sorry for the delay (I have been a few days off). Updated README now using `Fixed` tag.